core/src/main/java/org/apache/iceberg/stats/StatsFiles.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iceberg.stats;
+
+import java.util.Map;
+import org.apache.iceberg.io.OutputFile;
+import org.apache.iceberg.relocated.com.google.common.collect.Maps;
+
+public final class StatsFiles {
+  private StatsFiles() {
+  }
+
+  public static WriteBuilder write(OutputFile outputFile) {
+    return new WriteBuilder(outputFile);
+  }
+
+  public static class WriteBuilder {
```

Comment:
Does it buy you anything to have this as a nested class? It's the only thing in the StatsFile class

Reply:
Consistency, see eg `org.apache.iceberg.parquet.Parquet.WriteBuilder`